### DISCUSSION TITLE
Results/Statistics Tab no longer displays my errors

### DISCUSSION THREAD

**Discussion by ahegidus** [dbeaver/dbeaver#21020] [Q&A] [answered]
### Description

This started happening recently despite me not changing any settings so I am reporting it as a bug.

I have no idea what steps to provide to recreate it? Some of my team member are saying theirs hasn't changed.

Here's a screenshot. It will _sometimes_ show my error in the bottom right, but sometimes this is just blank. I really miss seeing the error in the entire results screen and it's becoming quite a hinderance. 

![Screenshot (103)](https://github.com/dbeaver/dbeaver/assets/87034584/2933a631-2583-4427-8213-da229c5fe718)

Is this a bug or a setting I can change?

### DBeaver Version

Community Version 23.1.5.202308201919

### Operating System

Windows

### Database and driver

Postgres 14

**Comment by E1izabeth** [comment]
Please, try pressing a little blue arrow on the left of this view

**Comment by E1izabeth** [comment, marked as answer]
![image](https://github.com/dbeaver/dbeaver/assets/28875055/78a90d11-61ae-4147-909d-553fa56cda11)

**Comment by ahegidus** [comment]
THANK YOU!!! I had tried so many things!!

**Comment by ahegidus** [comment]
That resolved it
